pkgs/development/python-modules/pyomo/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pyutilib
+, appdirs
+, ply
+, six
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "pyomo";
+  version = "5.6.1";
+
+  src = fetchPypi {
+    pname = "Pyomo";
+    inherit version;
+    sha256 = "449be9a4c9b3caee7c89dbe5f0e4e5ad0eaeef8be110a860641cd249986e362c";
+  };
+
+  checkInputs = [ nose ];
+  propagatedBuildInputs = [
+    pyutilib
+    appdirs
+    ply
+    six
+  ];
+
+  checkPhase = ''
+    rm pyomo/bilevel/tests/test_blp.py \
+       pyomo/version/tests/test_installer.py
+    nosetests
+  '';
+
+  meta = with lib; {
+    description = "Pyomo: Python Optimization Modeling Objects";
```

Comment:
The package name should not be included in the description: https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes

Reply:
This PR looks like it was already merged. I can still make the quick fix if needed.

Reply:
Not needed, but please adhere to the contributing guidelines next time!